README.md

```diff
@@ -82,11 +82,44 @@ def view_requiring_flag(request):
 def other_view(request):
     return HttpResponse('flag was not set')
 
-@flag_required('MY_FLAG', fallback_view=other_view)
+@flag_required('MY_FALLBACK_FLAG', fallback_view=other_view)
 def view_with_fallback(request):
     return HttpResponse('flag was set')
 ```
 
+For more complex use a `flag_check` decorator is provided that can be used to check for a particular value of a flag, with an optional fallback.
+
+```python
+from flags.decorators import flag_check
+
+@flag_check('MY_FLAG', True)
+def view_requiring_flag(request):
+    retrun HttpResponse('flag was set')
+
+@flag_check('MY_OTHER_FLAG', False)
+def view_when_flag_is_not_set(request):
+    retrun HttpResponse('flag was set')
+
+def other_view(request):
+    return HttpResponse('flag was not set')
+
+@flag_required('MY_FALLBACK_FLAG', True, fallback=other_view)
+def view_with_fallback(request):
+    return HttpResponse('flag was set')
+```
+
+For URL handling, there is `flagged_url()` which can be used in place of Django's `url()`. **Note**: it will not work for `include()` urls.
+
+```python
+from flags.urls import flagged_url
+
+urlpatterns = [
+    flagged_url('MY_FLAG', r'^my-url$', view_requiring_flag, condition=True),
+    flagged_url('MY_FALLBACK_FLAG', r'^my-url$', view_with_fallback, 
+                condition=True, fallback=other_view)
+]
```

Comment:
This documentation is a bit misleading, since both of these views are defined with the same URL pattern. If `MY_FALLBACK_FLAG` is set, but `MY_FLAG` is not, this will return a 404, won't it, because the Django URL dispatcher will match against the first entry, and never have a chance to match against the second one?